### PR TITLE
Inspector update [JIRA: RCS-139]

### PR DESCRIPTION
**Pull request by kuenishi**
Partly addresses #1013 and additional info when listing `moss.buckets`

### CODE REVIEW

File: priv/tools/internal/riak_cs_inspector.erl [hunk 1]
```diff
@@ -259,10 +275,24 @@ print_manifest_summary(_RiakcPid, _Bucket, SiblingNo, {tombstone, {_Bucket, Key}
     io:format("~-32s: ~-8B ~-16s ~-32s ~16B ~-16s~n",
               [Key, SiblingNo, tombstone, tombstone, 0, tombstone]);
 print_manifest_summary(RiakcPid, Bucket, SiblingNo, M) ->
+    State = m_attr(state, M),
+    case proplists:get_value(multipart, m_attr(props, M)) of
```
Comment (shino) [inline priv/tools/internal/riak_cs_inspector.erl:279]: `props` may be `undefined` :cry:

File: priv/tools/internal/riak_cs_inspector.erl [hunk 2]
```diff
@@ -167,14 +167,20 @@ cs_bucket_info(RiakBucket, KnownNames) ->
 
 -spec list_objects(pid(), string()) -> any().
 list_objects(RiakcPid, "moss.buckets" = Bucket)->
-    io:format("~-64..=s ~-8..=s: ~-40..=s~n",
-              ["CS Bucket Name ", "Sibl. ", "Owner Key "]),
+    io:format("~-64..=s ~-8..=s ~-40..=s~-20..=s~n",
+              ["CS Bucket Name ", "Sibl. ", "Owner Key ", " Last Updated "]),
```
Comment (shino) [inline priv/tools/internal/riak_cs_inspector.erl:171]: Bucket ACL for "Last Update" is correct for bucket creation and deletion.
I don't know it's correct or not in updating bucket ACL or bucket Policy.

Using `X-Riak-Last-Modified` is better?

**Comment by shino** [comment]
It's nice idea to show multipart parts explicitly :+1: 
By the formatting by this PR, the part number does not shown if key string is long (and truncated).
How about adding column(s) that shows the type and part number:
- normal upload or multipart upload
- part number for part manifest

File: priv/tools/internal/riak_cs_inspector.erl (hunk 2 above)
Comment (kuenishi) [inline priv/tools/internal/riak_cs_inspector.erl:171]: Yes, +1 to that. Maybe I can work on that later, or not (shrug)

**Comment by shino** [comment]
This PR is tagged to milestone 2.0.0 but merge target branch is `release/1.5`. New PR for `develop` needed?

**Comment by kuenishi** [comment]
Multipart printing was fixed by #1084 (RCS-132) .

**Pull request closed without merging**